docs/Configuration.md

```diff
@@ -171,3 +171,49 @@ nav_order: 15
 | spark.gluten.supported.scala.udfs                                                     || Supported scala udf names.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | spark.gluten.ui.enabled                                            | true              | Whether to enable the gluten web UI, If true, attach the gluten UI page to the Spark web UI.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 
+## Parquet write configurations
```

Comment:
The markdown file is auto generated from the configuration file after https://github.com/apache/incubator-gluten/pull/10190 we may need to move this section out to a new file as followup

Reply:
It's better to use the single file and append parquet write config after auto generation.

Reply:
> the UTs of this PR passed? Not as expected.

@yikf Yes, the UT is passed